### PR TITLE
Report Improved : from list reports to card reports.

### PR DESCRIPTION
**Pull request by shivamsherje**
Added card structure 
changes in styling and HTML

### CODE REVIEW

File: theme/moove/scss/default.scss [hunk 1]
```diff
@@ -16,4 +16,37 @@
 @import "moove/mydashboard";
 @import "moove/frontpage";
 @import "moove/profile";
-@import "moove/enrol";
\ No newline at end of file
+@import "moove/enrol";
+
```
Comment (AishwaryShrivastav) [inline theme/moove/scss/default.scss:20]: Remove these extra spaces

File: theme/moove/scss/default.scss [hunk 2]
```diff
@@ -16,4 +16,37 @@
...
+    justify-content: center;
+    text-align: center;
+    box-shadow: rgba(247, 147, 17, 0.12) 0px 2px 4px 0px, rgba(247, 147, 17, 0.32) 0px 2px 16px 0px;
+    transition: box-shadow 0.3s ease; 
+  }
+  
+  .report-card:hover {
+    box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.1); 
+  }
+      body.pagelayout-incourse #region-main a:not(.btn),body.pagelayout-incourse .section-navigation>a:not(.btn) {
```
Comment (AishwaryShrivastav) [inline theme/moove/scss/default.scss:36]: Missing proper spacing

**Review by AishwaryShrivastav — COMMENTED** [review]
@shivamsherje Please make the changes .
Every code changes should properly aligned

File: theme/moove/scss/default.scss (hunk 1 above)
Comment (shivamsherje) [reply theme/moove/scss/default.scss:20]: I have added this spaces for separation

**Pull request closed without merging**